dataland-community-manager/communityManagerOpenApi.json

```diff
@@ -1352,7 +1352,10 @@
       "CompanyRole": {
         "type": "string",
         "enum": [
-          "CompanyOwner"
+          "CompanyOwner",
+          "DataUploader",
```

Comment:
Ich glaube gemäß der Ticket-Description müsstest du die Uploader Rolle nochmal aufsplitten in externe Uploader und company-interne Uploader (siehe diese Tabelle in der Ticket-Beschreibung).
Denn company-interne Uploader sollen scheinbar noch "request unlimited datasets" dürfen, was die externen Uploader nicht dürfen sollen (siehe "y" und "x" ganz rechts in der Spalte).

Reply:
Hat sich erledigt weil Andreas gesagt hat, dass wir die Rolle als eine einzelne behandeln sollen.